src/App/main.cpp

```diff
@@ -60,6 +59,7 @@ int parseArgs(int argc, char **argv) {
     return 0;
 }
 
+#include <chrono>
```

Comment:
Why chrono?

Reply:
didn't mean to leave that in there. I'll take it out